cypress/integration/deals.js

```diff
@@ -0,0 +1,341 @@
+/* eslint-disable linebreak-style */
+const { login } = require('../../helpers/login-helper');
+const { generateName, generateRandomNum } = require('../../helpers/name-helper');
+
+context('Deals', () => {
+  describe('Deals UI', () => {
+    let userSessionToken;
+
+    before(() => {
+      userSessionToken = login('kargoqa@gmail.com', 'K@rgo123!');
+    });
+
+    // This beforeEach is needed because when a subsequent tests hits
+    // the same URL, cookies is not properly retained
+    beforeEach(() => {
+      cy.setCookie('kauth_access', userSessionToken);
+    });
+    const dealData = {
+      formatType: 'Bottom Banner',
+      executionName: 'AbedEXECUTION  28-6 8.51',
+      rate: generateRandomNum(200),
+      discount: generateRandomNum(10),
+      targetSpend: generateRandomNum(200),
+      bidderName: '1072 Charo',
+      advertiser: '10aug2017adver -dev',
+      priority: 'Highest',
+    };
+    let dealID;
+    let citadelValue;
+    // let masValue; temporarily until MAS is working
```

Comment:
see if you can enable mas targeting for this test cast (uncomment mas related code and try to get it to work).